ddtrace/contrib/anthropic/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+Do later.
```

Comment:
We'll need a brief documentation about the Anthropic integration, including:
- Summary about what it traces, similar to the OpenAI integration docs
- Prompt and completion sampling (similar to OpenAI docs)
- streamed responses support section (once we add that in, not necessarily for this PR)
- Enabling (similar to openai docs)
- Global configuration (service, span char limit, pc sample rate)
- Instance configuration (similar to openai docs)

Reply:
done